### ISSUE TITLE
Teletubbies: Accessibility - the user is unable to wake up the teletubby using the tabbing or screenreader functions.

### ISSUE THREAD

**Issue by AnneMcL** [FolaOlakunbi/Misc-Games-#2]
Both with screenreader and tabbing function, the user is unable to waken Tinky winky or any other Teletubbies 

1. Selected play to kick off the game https://tinyurl.com/4yzwud7t. Note that the user is prompted to waken the tele tubby. 
2. Using tabbing function or using the screenreader, attempt to waken the teletubbie (a typical user using a mouse can easily waken the teletubbie. 
3. Expected results: User should be able to waken teletubbie. Actual results: the teletubby is bypassed and the user is unable to waken any of the Teletubbies and progress with the game.
